### PR TITLE
Added travis config

### PR DESCRIPTION
**Pull request by jfeng-dev**
Initial basic travis config file.

### CODE REVIEW

**Comment by ZurabWeb** [comment]
@j-feng13 Travis don't like its config changed :angry:

**Comment by jfeng-dev** [comment]
I copied it from Swivel and removed some of the lines that weren't being used for this project. Did I set it up incorrectly?

**Comment by jrbasso** [comment]
@j-feng13 It seems related with the phpunit/composer configuration and not with the travis itself.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: php
+php:
+  - 5.4
```
Comment (jrbasso) [inline .travis.yml]: @j-feng13 We don't need to support PHP 5.4 for vanilla.

**Pull request merged**